src/webroot/js/cake-core.js

```diff
@@ -0,0 +1,25 @@
+//$(document).ready(function () {
```

Comment:
Out of my comment in #7511:

As the jQuery ready function is hard to rebuild in native javascript (some different events and window.onload is not called with CSP), I just add the js code in the root scope.
The idea was to force (automatically or manual by the user) to load the js file at the end of the website.